api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/AlgorithmConfigurations.scala

```diff
@@ -135,9 +135,13 @@ trait AlgorithmConfigurations extends ConfigurationsBase {
       ("H2OXGBoost", classOf[XGBoostParameters], "H2OTreeBasedSupervisedAlgorithm", Seq(withDistribution)),
       ("H2OGBM", classOf[GBMParameters], "H2OTreeBasedSupervisedAlgorithm", Seq(withDistribution)),
       ("H2ODRF", classOf[DRFParameters], "H2OTreeBasedSupervisedAlgorithm", Seq(withDistribution)),
-      ("H2OGLM", classOf[GLMParameters], "H2OSupervisedAlgorithm", Seq(withFamily)),
-      ("H2OGAM", classOf[GAMParameters], "H2OSupervisedAlgorithm", Seq(withFamily)),
-      ("H2ODeepLearning", classOf[DeepLearningParameters], "H2OSupervisedAlgorithm", Seq(withDistribution)),
+      ("H2OGLM", classOf[GLMParameters], "H2OSupervisedAlgorithmWithFoldColumn", Seq(withFamily)),
+      ("H2OGAM", classOf[GAMParameters], "H2OSupervisedAlgorithmWithFoldColumn", Seq(withFamily)),
+      (
+        "H2ODeepLearning",
```

Comment:
this also seems to be on multiple lines while previous items are one liners

Reply:
The SW uses spotless plugging during the gradle build. There is a rule for the maximum line size.  This line is above the limit and when I run `./gradlew spotlessApply` to apply the rules, the code was re-formatted to multiple lines.